### DISCUSSION TITLE
Accept package and source agreements

### DISCUSSION THREAD

**Discussion by fergusstrachan** [svrooij/WinTuner#52] [Ideas]
Hey,
Just come across a slight issue with the detection scripts.
Installed winget via Intune and installing some apps created by Wintuner afterwards.

Intune is giving me Error code: "0x87D1041C The application was not detected after installation completed successfully" but it is installed.

Running winget locally prompts me to accept the agreement etc etc the first time. Could this be why the detection script is failing? Perhaps just needs the --accept-package-agreements and --accept-source-agreements switches added.

I'm trying it out but thought I'd post something prior in case it's known or could be a good discussion.

Ferg.

**Comment by fergusstrachan** [comment]
It wasn't that. :-)

**Comment by svrooij** [comment]
You might be onto something, the detection scripts use winget to detect apps. Never tested it without accepting the license.

Can you tell me more about which package it's about?

**Reply by fergusstrachan** [reply]
It's "Adobe Acrobat Reader DC (64-bit)" package.

I'll need to set it up and install again to test it, but I'm pretty sure the issue was Wintuner packaged it and installed with msiexec, and running "winget list" on the device gave me the MSI code rather than the winget ID, so the detection script didn't work - it was looking for the store ID.

Hope this makes sense.

I'm having lots of fun playing about with this.

**Reply by fergusstrachan** [reply]
Btw I'm sure it wasn't the switches in my 1st post. The one I was prompted for was the source agrrement, and the switch for that is actually in your scripts.
